app/policies/kudo_policy.rb

```diff
@@ -0,0 +1,34 @@
+class KudoPolicy < ApplicationPolicy
+  def initialize(employee, kudo)
+    @employee = employee
+    @kudo = kudo
+  end
+
+  def index?
```

Comment:
sam thing here - in general in the policy you only place methods that you want to overwrite

Reply:
Methods which aren't overwritten - removed.